src/pages/getStarted/components/constants.ts

```diff
@@ -156,7 +158,44 @@ if __name__ == '__main__':
   with fp.Client('${serverSdkKey}', config) as client:
     user = fp.User()
     ${userWithCode}
-    val = client.value('${toggleKey}', user, default=${returnType === 'boolean' ? 'False' : ''}${returnType === 'string' ? 'not connected' : ''}${returnType === 'number' ? '-1' : ''}${returnType === 'json' ? '{}' : ''})  
+    val = client.value('${toggleKey}', user, default=${returnType === 'boolean' ? 'False' : ''}${returnType === 'string' ? '\'not connected\'' : ''}${returnType === 'number' ? '-1' : ''}${returnType === 'json' ? '{}' : ''})  
+`
+    }
+  ];
+};
+
+export const getNodeCode = (options: IOption) => {
+  const { intl, serverSdkKey, userWithCode, returnType, toggleKey, remoteUrl } = options;
+
+  return [
+    {
+      title: intl.formatMessage({id: 'getstarted.node.first.step'}),
+      code: 'npm install featureprobe-server-sdk-node --save'
+    },
+    {
+      title: intl.formatMessage({id: 'getstarted.node.second.step'}),
+      code:
+`import { FeatureProbe, FPUser } from 'featureprobe-server-sdk-node'
+
+const fpClient = new FeatureProbe({
+    serverSdkKey: '${serverSdkKey}',
+    remoteUrl: '${remoteUrl}',
+    refreshInterval: 2000,
+})
+await fpClient.start();  // if you want a time limit for the initialization process, set 'startWait' as the timeout milliseconds
+`
+    },
+    {
+      title: intl.formatMessage({id: 'getstarted.node.third.step'}),
+      code:
+`const user = new FPUser()${userWithCode};
+const toggleValue = fpClient.${returnType}Value('${toggleKey}', user, ${returnType === 'boolean' ? 'false' : ''}${returnType === 'string' ? '\'not connected\'' : ''}${returnType === 'number' ? '-1' : ''}${returnType === 'json' ? '{}' : ''});
+`
+    },
+    {
+      title: intl.formatMessage({id: 'getstarted.fourth.step'}),
```

Comment:
intl key `getstarted.fourth.step` lost a letter `C` in en-US.json, help us to correct it in this pr.